app/views/home/index.html.erb

```diff
@@ -8,9 +8,7 @@
               <img src="/assets/panel-image-1.png" alt="" />
             </a>
           </div>
-
           <div class="col-md-5">
-
           <% if user_signed_in? %>
```

Comment:
can you indent this code? Maybe is already indented, but this file is so big that i think i got lost.